File: HumanitarianAssistance.Entities/Models/Common/FileManagementModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace HumanitarianAssistance.ViewModels.Models.Common
{
    public class FileManagementModel : BaseModel
    {
        public string FileType { get; set; }
        public string FileName { get; set; }
        public long FileSize { get; set; }
        public int PageId { get; set; }
        public long RecordId { get; set; }
        public string FilePath { get; set; }
    }

    public class FileModel
    {
        public int PageId { get; set; }
        public long? RecordId { get; set; }
        public long? DocumentFileId { get; set; }
    }

    public class FileListModel
    {
        public string FileName { get; set; }
        public string FileSignedURL { get; set; }
        public string FilePath { get; set; }
        public long? DocumentFileId { get; set; }
    }
}

File: HumanitarianAssistance.WebApi/OldUI/src/app/shared/FileManagement/file-management-model.ts
export interface FileManagementModel {
    FileName: string;
    File: any;
    FileType: string;
    FileLength: number;
}

export interface FileModel {
    FileType: string;
    FileName: string;
    FileSize: number;
    PageId: number;
    FilePath: string;
    RecordId: number;
    DocumentTypeId?: number;
    DocumentFileId?: number;
}

export interface UploadModel {
    PageId: number;
    EntityId: number;
    File: any;
    DocumentTypeId?: number;
    DocumentFileId?: number;
}
